pkg/tanka/tanka.go

```diff
@@ -0,0 +1,85 @@
+// The tanka package allows to use most of Tankas features available on the
```

Comment:
```suggestion
// The tanka package allows to use most of Tanka's features available on the
```

Extremely minor nit, sorry